src/pages/Fox/components/FoxGovernanceProposal.tsx

```diff
@@ -0,0 +1,77 @@
+import { Box, Card, CardBody, Flex, Link, Progress, Tag, Text } from '@chakra-ui/react'
+import { foxAssetId } from '@shapeshiftoss/caip'
+import { Amount } from 'components/Amount/Amount'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+type FoxGovernanceProposalProps = {
+  title: string
+  description: string
+  options: string[]
+  results: number[]
+  totalScore: number
+  link: string
+}
```

Comment:
suggestion: leverage `Proposal` type I.e with `Pick<T, U>` currently, or even better if we request only the fields we want (see https://github.com/shapeshift/web/pull/7952#discussion_r1806202732), then we can use the full `Proposal` type here without any picking or ommitting, or perhaps pass `proposal: Proposal` if we prefer